pallets/gear/src/lib.rs

```diff
@@ -1648,6 +1646,45 @@ pub mod pallet {
 
             Ok(())
         }
+
+        /// Transfers value from terminated or exited program to its living inheritor.
+        ///
+        /// Inheritor can be an active program or original user.
+        ///
+        /// `depth` parameter is how far to traverse to inheritor.
+        #[pallet::call_index(8)]
+        #[pallet::weight(<T as Config>::WeightInfo::transfer_value_to_inheritor(depth.get()))]
+        pub fn transfer_value_to_inheritor(
+            origin: OriginFor<T>,
+            program_id: ProgramId,
+            depth: NonZeroU32,
```

Comment:
I'd avoid this not to mislead caller clients like js code, zero depth is just no-op operation